pepper-apis/studybuilder-cli/studies/osteo/i18n/en.conf

```diff
@@ -31,6 +31,28 @@
     "err_international_child": "Currently, Count Me In is open only to patients in the United States or Canada. If your child does live or is treated in the United States or Canada, please reach out to us at <a href=\"mailto:info@joincountmein.org\" class=\"Link\">info@joincountmein.org</a>.",
     "err_need_parental": "<span class=\"bold\">In order to participate in the project, a parent needs to help you.</span> When your parent is with you, click back and select \"My child has been diagnosed\" and complete the registration together.",
     "err_need_self": "In order to participate in the project, your child must consent and register on their own."
+
```

Comment:
just a update from osteo spanish translations so that generic patches dont complain.